LithoStrings/Classes/NSAttributesReturnFunctions.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  NSAttributesReturnFunctions.swift
+//  LithoStrings
+//
+//  Created by Remmington Damper on 1/26/22.
+//
+
+import Foundation
+import UIKit
+
+    public func returnForegroundColor(color: UIColor) -> [NSAttributedString.Key: Any] {
```

Comment:
let's do 

```suggestion
public func foregroundColorAttributes(with color: UIColor)
```

And apply that to the rest of the functions as well